extensions/markdown-language-features/src/languageFeatures/pathCompletions.ts

```diff
@@ -325,7 +324,7 @@ export class MdVsCodePathCompletionProvider implements vscode.CompletionItemProv
 
 	private resolvePath(root: vscode.Uri, ref: string): vscode.Uri | undefined {
 		try {
-			if (root.scheme === Schemes.file) {
```

Comment:
Will fix the root cause of this later. The issue is that `Scheme.file` has a trailing `:`